test/src/api/routes/test-edition-group.js

```diff
@@ -70,6 +70,17 @@ describe('GET /EditionGroup', () => {
 		);
 	 });
 
+	it('should return list of relationships of an edition group', async function () {
+		const res = await chai.request(app).get(`/edition-group/${aBBID}/relationships`);
+		expect(res.status).to.equal(200);
+		expect(res.body).to.be.an('object');
+		expect(res.body.relationships).to.be.an('array');
+		expect(res.body).to.have.all.keys(
+			'bbid',
+			'relationships'
```

Comment:
I'd like to see more tests for the relationships. You do test that you get an array, but not for the structure of a relationship in that array.

Your will need, in new tests, to add a relationship to the test data in the database. (Perhaps you can create a helper in create-entities.js)
Then query the api endpoint and check that the response has all the expected keys for a relationship.
You'll also want to compare values for each key.
You'll need a test for each of these use cases: backward/forward relationship and linkphrase, two different relationshipTypes, two different targetEntityTypes.
And finally a test to make sure you do get multiple relationships (in a test, create two relationships an ensure relationships.length === 2

Reply:
I've added the necessary code to create-entities, so you should now be able to add those tests.

Reply:
@akhilesh26 Circling back on this, I'd like to see a separate test each for aliases, identifiers and relationships that makes sure their structure is correct.
You would only need to have those tests for one entity type.

For example, ensure that `/author/{bbid}/aliases` returns and object, that result.aliases is an array, and that the first item in that array is an object with the keys aliasLanguage, name, primary and sortName, and finally check the type of each property (aliasLanguage, name and sortName are strings, primary is a boolean).
Do the same for relationships and identifiers and we'll be assured the structure is stable, and users can expect exactly that structure.